### ISSUE TITLE
Heatmap yaxislabel can not be set

### ISSUE THREAD

**Issue by benediktwetzel** [britecharts/britecharts#899]
### Motivation
I want to change the YAxisLabels of the heatmap. I'm using Vue and tried it with: 

```ts
this.heatMap = new heatmap().yAxisLabels(['A', 'B', 'C', 'D', 'E', 'F', 'G'])
```

### Expected Behavior
The labels set on the Y-axis are displayed.

### Difference from Current Behavior
The default labels are displayed ('Mo', 'Tu', 'We', 'Th', 'Fr', 'Sa', 'Su'). However, `heatmap.yAxisLabel()`' returns the new values (['A', 'B', 'C', 'D', 'E', 'F', 'G'])`. 

### A minimal code example:

```ts
@Component({
  components: {}
})
export default class CorrelationHeatmap extends Vue {
    private heatMap!: any;
    private container!: d3.Selection<any, any, HTMLElement, undefined>;

    private createData () {
      const data: Array <{ 'day': number; 'hour': number; 'value': number }> = []
      for (let i = 0; i < 7; i++) {
        for (let j = 0; j < 24; j++) {
          const val = Math.floor(Math.random() * Math.floor(100))
          data.push({
            day: i,
            hour: j,
            value: val
          })
        }
      }
      return data
    }

    mounted () {
      this.container = d3select('.correlation-heatmap')
      this.container.html('')
      const values = this.createData()
      this.heatMap = new heatmap()
        .yAxisLabels(['A', 'B', 'C', 'D', 'E', 'F', 'G'])
      this.container.datum(values).call(this.heatMap)
    }
}
``` 

What could be the problem?

**Comment by Golodhros** [comment]
Hi @benediktwetzel, thanks for opening this issue!

This is a bug indeed, it was caused by an issue when merging a conflict. Here is the fix: https://github.com/britecharts/britecharts/pull/900

**Comment by Golodhros** [comment]
The fix should be ready on 2.17.6

**Comment by benediktwetzel** [comment]
It works for me. Thanks a lot!
